config/settings.yml

```diff
@@ -6,5 +6,6 @@ doorkeeper:
   administrators:
     - rainbowdash
 secret_key_base: CHANGE ME
+sentry_dsn: hahano
```

Comment:
Ah, right.  I hadn't thought about it not being set in production.yml.  I'll update.